Quran/SettingsCreators.swift

```diff
@@ -62,13 +62,13 @@ class NavigationSettingsCreators: SettingsCreators {
             }
             Analytics.shared.review(automatic: false)
         }
-        let email = Setting(name: l("email_us"), image: #imageLiteral(resourceName: "email-outline")) { vc in
+        let email = Setting(name: l("contact_us"), image: #imageLiteral(resourceName: "email-outline")) { vc in
             if MFMailComposeViewController.canSendMail() {
                 let mail = MFMailComposeViewController()
                 mail.navigationBar.tintColor = .white
                 mail.mailComposeDelegate = self.parentController
-                mail.setToRecipients(["ios@quran.com"])
-                mail.setSubject("Feedback About Quran Application")
+                mail.setToRecipients([l("email_recipeint")])
+                mail.setSubject(l("email_subject"))
```

Comment:
Moved to Localization file

Reply:
Thanks for the update. But I wouldn't do that. Because I don't want to get an e-mail with a Russian subject. Also, ios@quran.com can't be translated. Those 2 things are a good candidate to be configurable. But since we don't have configuration yet, I would leave them as hard-coded values.

Reply:
Updated.